### ISSUE TITLE
Provide screenshot suggestions

### ISSUE THREAD

**Issue by desiderantes** [elementary/houston#639]
Generating screenshots for applications would be a really nice feature, and creating a safe default picture in case there's something wrong with other screenshots.

There's discussion about this feature [here](http://curiositydrivendevelopment.blogspot.com/2018/07/gnome-application-screenshots-with.html), but the implementation discussed has the following problems:

- Depends on X11 tooling (I have no info on Wayland alternatives)
- Provides barebones GTK+ theming (I guess this is where Houston would shine)
- Does not deals with apps with long startup time, apps with initial steps before the main window, nor apps with required configuration

**Comment by cassidyjames** [comment]
We've discussed this before, but the result is usually the same: Auto-generating screenshots will almost always yield a worse result than the developer (who knows their application and which screens are useful to users) taking and picking them. Given that, I don't know that the tooling required to automatically take screenshots (plus the edge cases it introduces) is worth investing time and effort into right now.

**Comment by danrabbit** [comment]
There's also the problem of apps being content-less. A screenshot of an empty Video player isn't very interesting. I'm leaning towards this being a "Won't fix" since it's far less effort to just instruct the developer to take a screenshot
